examples/03_connectivity/plot_compare_resting_state_decomposition.py

```diff
@@ -90,7 +90,7 @@
 from nilearn.image import index_img
 
 # Selecting specific maps to display: maps were manually chosen to be similar
-indices = {dict_learning: 25, canica: 33}
+indices = {dict_learning: 24, canica: 32}
 # We select relevant cut coordinates for displaying
 cut_component = index_img(components_imgs[0], indices[dict_learning])
 cut_coords = find_xyz_cut_coords(cut_component)
```

Comment:
@KamalakerDadi , on this example, some of the visualizations of regions are broken: the contours are present, but not filled. Do you have an idea of why that might be the case?
https://3567-1235740-gh.circle-artifacts.com/0/home/circleci/project/doc/_build/html/auto_examples/03_connectivity/plot_compare_resting_state_decomposition.html#sphx-glr-auto-examples-03-connectivity-plot-compare-resting-state-decomposition-py
Do you have

Reply:
Well yeah I saw that and needs to be investigated.

Reply:
Does this looks good now ?

https://3645-1235740-gh.circle-artifacts.com/0/home/circleci/project/doc/_build/html/auto_examples/03_connectivity/plot_compare_resting_state_decomposition.html#sphx-glr-auto-examples-03-connectivity-plot-compare-resting-state-decomposition-py

Reply:
It is OK, but the CanICA components is probably not 32.
Also the images are extremely *smooth*.

Reply:
>Also the images are extremely smooth.

Is this a bad sign or good sign ?

Reply:
Neither, but qualitatively it gives the impression of slightly too much smoothing on the data.

Reply:
Are the images downloaded smoothed? @emdupre @illdopejake , do you know?

Reply:
AFAIK we did not additionally smooth them (please correct me if that's incorrect @illdopejake  !), but they were smoothed prior to distributing as OpenNeuro derivatives. From the [derivatives description](https://openneuro.org/datasets/ds000228/versions/1.0.0/file-display/derivatives:README):

> All data were smoothed using a Gaussian filter (5mm kernel).

Reply:
> AFAIK we did not additionally smooth them (please correct me if that's incorrect @illdopejake !)
That's correct, we did not do any additional smoothing. It's possible that we could reach out to the Saxe lab to see if they would mind posted the unsmoothed data?

Reply:
Any news on this non-smoothed data sharing aspect ?

Reply:
I think it would be easiest reprocess them ourselves, but not sure if @illdopejake is in contact with Saxe lab.

If we do reprocess them ourselves I can run them through fMRIPrep on Compute Canada, but I need to set up an account. Not sure if you already have one we could submit through, Jake ? I have an fMRIPrep singularity image....